examples/sudoku/ts/src/sudoku-zkapp.ts

```diff
@@ -106,32 +97,32 @@ class SudokuSnapp extends SmartContract {
 const Local = Mina.LocalBlockchain();
 Mina.setActiveInstance(Local);
 const account1 = Local.testAccounts[0].privateKey;
-const account2 = Local.testAccounts[1].privateKey;
 
-const snappPrivkey = PrivateKey.random();
-let snappAddress = snappPrivkey.toPublicKey();
+const zkappKey = PrivateKey.random();
```

Comment:
slight preference for `zkAppPrivateKey` b/c of the casing of zkApp and clarity of name for the private key.